### ISSUE TITLE
Bottom of search results paging

### ISSUE THREAD

**Issue by dbranchini** [sul-dlss/earthworks#1380]
Paging at the bottom of the search results isn't appearing as links. See screenshot.

**Currently:** 
<img src="https://github.com/user-attachments/assets/3aede27b-cd20-41b5-86c7-93b3e88dc5b2" width="400">

**As designed:**
<img src="https://github.com/user-attachments/assets/2cd1e20e-da81-44a1-a429-05a5ddad0675" width="300">

Also note, we're getting an **accessibility error** on the use of `aria-current="page"` due to double quotes. I'm also concerned that we might be mixing and matching two different accessibility approaches creating duplication of information. Note the use of both `aria-current="page"` and `aria-label="Current Page, Page 4"`. See screenshot.

<img src="https://github.com/user-attachments/assets/30ebb61d-7da5-41be-94a2-637d64fd1da7" width="400">

The original references to paging and accessibility are in this ticket - https://github.com/sul-dlss/earthworks/issues/1170.

**Comment by dnoneill** [comment]
closed by 
#1388 
https://github.com/projectblacklight/blacklight/pull/3316

**Comment by dnoneill** [comment]
Fixed with #1388 and https://github.com/projectblacklight/blacklight/pull/3316
